_posts/2022-02-23-bio-formats-6-9-0.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: Release of Bio-Formats 6.9.0
+intro-blurb: The OME team is pleased to announce the release of Bio-Formats 6.9.0
+---
+
+Today we are releasing Bio-Formats 6.9.0. This new release contains several file format updates and security fixes.
+
+New file formats:
+
+* Leica LOF
+   - added support for reading and writing Leica LOF files
+     This functionality was implemented with thanks to Leica Microsystems
+
+* Leica XLEF
+   - added support for reading and writing Leica XLEF files
+     This functionality was implemented with thanks to Leica Microsystems
```

Comment:
Community contributions are in bracket e.g. ``(thanks to Nils Gladitz)``. Leica contribution is a sentence. This could potentially be misinterpreted.